packages/esm-patient-chart-app/src/visit/visits-widget/past-visits-components/visits-table/visits-table.component.tsx

```diff
@@ -139,11 +139,11 @@ const VisitTable: React.FC<VisitTableProps> = ({ showAllEncounters, visits, pati
   const tableRows = useMemo(() => {
     return paginatedVisits?.map((encounter) => ({
       ...encounter,
-      datetime: formatDatetime(parseDate(encounter?.datetime)),
-    }));
+      datetime: formatDatetime(parseDate(encounter.datetime)),
+    }));    
   }, [paginatedVisits]);
 
-  const handleEncounterTypeChange = ({ selectedItem }) => setFilter(selectedItem);
+  const handleEncounterTypeChange = ({ selectedItem }) => setFilter(selectedItem.value);
```

Comment:
Hello @jwnasambu do we need this change? this change is restricting the filter to work.